Confluence Phase-2 (#88)
mattermost/mattermost-plugin-confluence: assets/templates/command/install_server.md

{{ .ConfluenceURL }} has been successfully added. To finish the configuration, add an Application Link in your Confluence instance following these steps:

1. Go to [**Settings > Applications > Application
   Links**]({{ .ConfluenceURL }}/plugins/servlet/applinks/listApplicationLinks)
2. Select **Create link**.
3. On the **Create Link** screen, select **External Application** and **Incoming** as
   `Application type` and `Direction` respectively. Select **Continue**.

> 4. On the **Link Applications** screen, set the following values:

Reviewer: ```suggestion
4. On the **Link Applications** screen, set the following values, then select **Continue**:
```

> **Name**: `Mattermost`

Reviewer: Please indent these 3 lines

**Redirect URL**: ```{{ .RedirectURL }}```.
**Application Permissions**: `Admin`

> Select **Continue**

Reviewer: I've incorporated this step into the line above. This line is no longer needed.

{{ .ConfluenceURL }} has been successfully added. To finish the configuration, add an Application Link in your Confluence instance following these steps:

> 1. Navigate to [**Settings > Applications > Application

Reviewer: What app is the user in when performing this step?

> 1. Navigate to [**Settings > Applications > Application

Author: logged in the server instance of confluence which could be something like this :- confluence.< Instance >.com and then user is required create a external link for Oauth purpose

   Links**]({{ .ConfluenceURL }}/plugins/servlet/applinks/listApplicationLinks)
2. Click **Create link**.
3. In **Create Link** screen, select **External Application** and **Incoming** as
   `Application type` and `Direction` respectively. Click **Continue**.
4. In **Link Applications** screen, set the following values:
**Name**: `Mattermost`
**Redirect URL**: ```{{ .RedirectURL }}```.
**Application Permissions**: `Admin`
Click **Continue**
5. Copy the `clientID` and `clientSecret` and add them into the plugin configuration.
6. Use the "/confluence connect" command to connect your Mattermost account with your
   Confluence account.

> 7. Use the "/confluence connect URL admin" for confluence admin only to save the token for the first time only.   

Reviewer: is this a slash command? Is it a one-time command?
is it available only to Confluence admins?

Author: yes this a slash command (This is just a slight variant of /confluence connect command) and this should be used only by the confluence admins for the first time as to create webhooks for confluence server we required admin permissions and token so with this command we securely save the admin token just for the creation and deletion of the webhooks for confluence server for every user

> 8. Click the "More Actions" (...) option of any message in the channel

Reviewer: ```suggestion
8. Select the **More** (...) option by hovering over a message.
```

Reviewer: What would the subsequent step be once the user has selected ...?

Author: Once the user is connected to a confluence instance using /confluence connect <url> command they can create subscription to the space/pages using /confluence subscribe command

>    (available when you hover over a message).

Reviewer: this line is no longer needed

Author: Just to confirm you are taking about point number 8 and line 17,18 which starts with Click the "More Actions " ...

Reviewer: Yes, thanks!
